opentelemetry-sdk/src/metrics/periodic_reader.rs

```diff
@@ -414,15 +402,6 @@ impl PeriodicReaderInner {
     }
 
     fn shutdown(&self) -> MetricResult<()> {
-        if self
-            .shutdown_invoked
```

Comment:
Just thinking - we don't need any check in periodically executed `collect_and_export` to _not_ do export if shutdown is invoked?

Reply:
after shutdown is invoked, there is no way to invoke collect or export methods (they are not public).
I am still checking for ways to better architect this - reader is cloneable today, which I am not sure if we need to. but requires more refactoring/design..